django/db/models/fields/related.py

```diff
@@ -34,6 +34,19 @@
 RECURSIVE_RELATIONSHIP_CONSTANT = 'self'
 
 
+def resolve_model_key(app_label, model_name, relation):
```

Comment:
I think you could use `django.db.models.migrations.operation.resolve_relation` for this purpose. Could be worth moving to `django.db.models.migrations.utils`.